src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/ConditionController.kt

```diff
@@ -0,0 +1,427 @@
+        responseCode = "404",
+        description = "The licence for this ID was not found.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+    ],
+  )
+  fun deleteAdditionalCondition(
+    @PathVariable("licenceId") licenceId: Long,
+    @PathVariable("conditionId") conditionId: Long,
+  ) {
+    return this.licenceConditionService.deleteAdditionalCondition(licenceId, conditionId)
+  }
+
+  @Tag(name = Tags.LICENCE_CONDITIONS)
+  @PutMapping(value = ["/id/{licenceId}/additional-conditions"])
+  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CVL_ADMIN')")
+  @Operation(
+    summary = "Update the set of additional conditions on the licence.",
+    description = "Update the set of additional conditions on the licence. " +
+      "This does not include accompanying data per condition. Existing conditions which appear on " +
+      "the licence but which are not supplied to this endpoint will be deleted. " +
+      "Requires ROLE_SYSTEM_USER or ROLE_CVL_ADMIN.",
+    security = [SecurityRequirement(name = "ROLE_SYSTEM_USER"), SecurityRequirement(name = "ROLE_CVL_ADMIN")],
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "Set of additional conditions updated",
+      ),
+      ApiResponse(
+        responseCode = "400",
+        description = "Bad request, request body must be valid",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorised, requires a valid Oauth2 token",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Forbidden, requires an appropriate role",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "The licence for this ID was not found.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+    ],
+  )
+  fun updateAdditionalConditions(
+    @PathVariable("licenceId") licenceId: Long,
+    @Valid @RequestBody
+    request: AdditionalConditionsRequest,
+  ) {
+    return licenceConditionService.updateAdditionalConditions(licenceId, request)
+  }
+
+  /**
+   * This functionality can be replaced once proper licence versioning is implemented
```

Comment:
Not a note for this PR or work to do now, but now that we have versioning, presumably we should review whatever this was referring to? 🤔

Reply:
I think this refers to us not setting the standard conditions via the frontend - we should be able to set/refresh them at various points in the licence life cycle purely from the backend

Reply:
I'll update the comment - not that it helps much but at least adds the context!